src/Command/CheckCommand.php

```diff
@@ -146,7 +146,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 null,
                 null,
                 $configuration,
-                null
+                null,
+                false
```

Comment:
The minimum required version from PHPStan should be also bumped, otherwise if < 0.11.18 is installed in an environment you are calling this method with too many parameters.